File: freecodecamp/daily-code-challenge/2026-02-14.py
def get_difficulty(track):
    score = 0
    
    for i in range(len(track)):
        if track[i] in ("L", "R"):
            if i > 0 and (
                (track[i] == "L" and track[i-1] == "R") or
                (track[i] == "R" and track[i-1] == "L")
            ):
                score += 15
            else:
                score += 5
    
    if score <= 100:
        return "Easy"
    elif score <= 200:
        return "Medium"
    else:
        return "Hard"

File: freecodecamp/daily-code-challenge/2026-02-14.js
function getDifficulty(track) {
  let score = 0;
  
  for (let i = 0; i < track.length; i++) {
    if (track[i] === "L" || track[i] === "R") {
      if (
        i > 0 &&
        ((track[i] === "L" && track[i - 1] === "R") ||
         (track[i] === "R" && track[i - 1] === "L"))
      ) {
        score += 15;
      } else {
        score += 5;
      }
    }
  }
  
  if (score <= 100) return "Easy";
  if (score <= 200) return "Medium";
  return "Hard";
}
